Triangulation_2/include/CGAL/Constrained_triangulation_2.h

```diff
@@ -881,10 +909,12 @@ intersect(Face_handle f, int i,
 // split constraint edge (f,i) 
 // and return the Vertex_handle of the new Vertex
 { 
-  std::cerr << "You are using an exact number types" << std::endl;
-  std::cerr << "using a Constrained_triangulation_plus_2 class" << std::endl;
-  std::cerr << "would avoid cascading intersection computation" << std::endl;
-  std::cerr << " and be much more efficient" << std::endl;
+  CGAL_warning_msg(false,
+  "You are using an exact number types\n"
+  "using a Constrained_triangulation_plus_2 class\n"
+  "would avoid cascading intersection computation\n"
+  " and be much more efficient\n");
+
```

Comment:
In #2752, @sloriot said that there should be a macro to allow the removal of that warning, if the macro is defined.

Reply:
CGAL_NO_WARNINGS works to discard it

Reply:
I agree, but I thought that Sébastien had something else in mind, a macro that would disable only that warning. @sloriot?